chia/wallet/wallet_state_manager.py

```diff
@@ -383,11 +383,10 @@ async def create_more_puzzle_hashes(
                     )
                 self.log.info(f"Done: {creating_msg} Time: {time.time() - start_t} seconds")
             await self.puzzle_store.add_derivation_paths(derivation_paths)
-            await self.add_interested_puzzle_hashes(
-                [record.puzzle_hash for record in derivation_paths],
-                [record.wallet_id for record in derivation_paths],
-            )
             if len(derivation_paths) > 0:
+                await self.wallet_node.new_peak_queue.subscribe_to_puzzle_hashes(
+                    [record.puzzle_hash for record in derivation_paths]
+                )
```

Comment:
it's not obvious why we want to subscribe only if we have any derivation_paths. Do we not need to subscribe to these puzzle hashes sometimes?

Reply:
See https://github.com/Chia-Network/chia-blockchain/pull/14187#discussion_r1062033968